peachjam/templates/peachjam/_timeline_events.html

```diff
@@ -11,7 +11,7 @@ <h5 class="mb-0">
               <span class="badge rounded-pill bg-primary">{% trans 'this version' %}</span>
             {% endif %}
             {% for event in history_item.events %}
-              {% if event.unapplied_amendment %}
+              {% if event.unapplied_amendment and history_item.date > latest_amendment_date %}
```

Comment:
Determining if it's an unapplied amendment is a business logic issue and should live in the python code, wherever `unapplied_amendment` is calculated.

Reply:
I agree that it should be after the latest expression date. Apologies if my write-up was unclear on that, Nick. I think I used 'document' where I should have used 'expression'. 

And yes, looks like `point_in_time_dates` has dates for actual expressions as opposed to just timeline dates.

I would probably change https://github.com/laws-africa/peachjam/blob/main/peachjam/views/legislation.py#L202 to include that extra condition.